lib/article-tile/index.js

```diff
@@ -1,12 +1,14 @@
 import React, { Component, PropTypes } from 'react';
 require('./style.css');
 import FadeImage from '../fade-image';
+import defaultImage from '../../src/constants/default-image';
```

Comment:
any reason this default image is not simply uploaded to CDN (e.g: S3) _once_?
does the `import` an image add it to the `bundle.js` or does it get shipped to S3 and loaded using "isomorphic fetch" ...?